db/rdbms/src/main/resources/mapper/UserTaskMapper.xml

```diff
@@ -214,6 +227,17 @@
         </foreach>
       </if>
 
+      <!-- tags -->
+      <if test="filter.tagOperations != null and !filter.tagOperations.isEmpty()">
+        AND EXISTS (Select * from ${prefix}USER_TASK_TAG t where t.USER_TASK_KEY =
+        ut.USER_TASK_KEY
+        <foreach collection="filter.tagOperations" item="operation">
+          AND t.TAG_VALUE
+          <include refid="io.camunda.db.rdbms.sql.Commons.operationCondition"/>
+        </foreach>
+        )
```

Comment:
❓ Why is a subselect required? In line 135, it already performs a left join with the `USER_TASK_TAG` table. So, why not simply use that joined table? It could be something like:
```sql
<if test="filter.tagOperations != null and !filter.tagOperations.isEmpty()">
  <foreach collection="filter.tagOperations" item="operation">
    AND t.TAG_VALUE
    <include refid="io.camunda.db.rdbms.sql.Commons.operationCondition"/>
  </foreach>
</if>
```